src/search/state/searchActions.ts

```diff
@@ -29,6 +29,7 @@ export const RECEIVE_EVIDENCES = 'RECEIVE_EVIDENCES';
 export const UPDATE_CLAUSES = 'UPDATE_CLAUSES';
 export const UPDATE_QUERY_STRING = 'UPDATE_QUERY_STRING';
 export const RESET = 'RESET';
+export const RESET_SEARCH_INPUT = 'RESET_SEARCH_INPUT';
```

Comment:
Not so much part of the review but more a question... (which would lead to a new Jira).

I understand that we don't want to reset to the initial state every time as we don't want to constantly rehydrate the state so resetting the search input instead is enough.

Currently a new search is triggered to load the search results when using the back button from an entry. This is not ideal as the search results are already in the state so could be loaded from there instead. However there is the issue of using the url to navigate, so I am wondering whether the results url could be saved in the state and compared to trigger a new request vs loading from the state? What do you think?